CONTRIBUTION.md

```diff
@@ -0,0 +1,89 @@
+3. It is a good practice to assign the issue to yourself to let others know you're working upon it.
+![Image](https://lh6.googleusercontent.com/MsF7-9lTnDgQsi888PqSFtWCOfoxnhDQVfNv6aem7AcEo06gDrCk-ISZ8C3VIe6AUfRfglRq3xgaHSzS2yfQHutWkbHingQswCYGpBqVOaa_WImECxYvKAZM-i6lr7HirFDAhA41)
+## Making changes to the codebase
+- Follow the instructions in the README.md to setup and install pbench.
+- Follow the instruction in the README.md to use web server (PBench Dashboard)
+- Save your changes by creating own local branches on git
+- While doing the changes, follow the style guides:
+  - JavaScript: https://github.com/airbnb/javascript
+  - React: https://github.com/airbnb/javascript/tree/master/react
+  - CSS: https://github.com/airbnb/javascript/tree/master/css-in-javascript
+ 
+## Add, Commit and Push
+- Follow these commands to push the changes to your branch. 
+```
+git add .
+git commit -m "Issue solved"
+git push origin branch_name 
+```
+ 
+## Conventions on commits, PRs, and overall git best practices. 
+- Commit messages should have a short description (50 - 70 characters) followed by a longer format description of the changes below if needed. You’ll also notice each line is formatted for a specific length in the longer format description. For example:
+Extend auditing to incoming, results, and users
+
+```
+The server audit is now applied to the incoming, results, and users
+directory hierarchies.  Any unpacked tar ball should now be compre-
+hensively checked to see that all is in the correct place.
+
+The test-20 unit test gold file holds an example of an audit report
+covering all the possible outputs it can emit.  Each unit test runs
+the report as well, and they have been updated accordingly.
+```
+- For more on best practices, check out this article for reference from time to time: https://www.git-tower.com/learn/git/ebook/en/command-line/appendix/best-practices
+## Opening a pull request
+1. **If there are multiple commits, Squash down the commits to one**
+2. Commit the changes.
+3. Click on New Pull Request
+4. Write appropriate Pull Request Title stating the fix.
+  a. Use present tense (ex. Fixes, Changes, Fixing, Changing..)
+5. References the issue that the PR is fixing with **“Fixes #issue_number”** in the description.
+6. Provide a detailed description (at least 50 - 70 characters) of the changes.(If UI, add screenshots or gif)
+7. Make sure that the branches can be automatically merged(otherwise rebase the PR with master) and then click : Create pull request .
+![Image](https://lh5.googleusercontent.com/V14SjFhimKYF1fH6TXMfaoZtDCj2ZH0d9USqe8YHyn0xOOVYekiXtx2CwdOQbSvxWPB6JVEfi4jSM_mjkSMaaI7voQNYQ8gDWntMhzCMbj3wrK3H4eCSEsdVq_XP_aZMdb5h9xU4)
+
+8. Assign the PR to yourself and add appropriate labels.
+9. Add “WIP” label if the work is still in progress.
```

Comment:
`WIP` means: I am not finished with this work, but plan to continue with future updates.

`Do Not Merge` means: Regardless of the state of this work, do not merge this into the tree.